zerver/webhooks/ifttt/view.py

```diff
@@ -16,13 +15,13 @@
 def api_iftt_app_webhook(
     request: HttpRequest,
     user_profile: UserProfile,
-    payload: Dict[str, Any] = REQ(argument_type="body"),
+    payload: WildValue = REQ(argument_type="body", converter=to_wild_value),
 ) -> HttpResponse:
-    topic = payload.get("topic")
-    content = payload.get("content")
+    topic = payload.get("topic").tame(check_anything)
+    content = payload.get("content").tame(check_anything)
```

Comment:
Why are these `check_anything`? I'm pretty sure they'll need to be strings in order to work...